starfish/image/_filter/richardson_lucy_deconvolution.py

```diff
@@ -14,7 +14,7 @@
 class DeconvolvePSF(FilterAlgorithmBase):
 
     def __init__(
-            self, num_iter: int, sigma: Number, clip: bool=False, **kwargs) -> None:
```

Comment:
Does this belong in this PR?

Reply:
Yea well it was something ambrose wanted me to change when I was working on the merfish pipeline and asking him about it. It is kinda out of the cope of this PR though if you want me to make a separate one for it.

Reply:
Yup, that would be ideal.